webapp/packages/core-root/src/SessionResource.ts

```diff
@@ -95,6 +99,30 @@ export class SessionResource extends CachedDataResource<SessionState | null> {
     return session;
   }
 
+  async touchSession(force?: boolean) {
+    if (!this.data?.valid || this.touchSessionTimer || !this.clientActivityService.isActive || !force) {
+      return;
+    }
+
+    const valid = await this.graphQLService.sdk.touchSession();
+
```

Comment:
this function will never be executed without `force=true`, `force` is `false` by default so any call `touchSession()` will do nothing